app/controllers/api/venues_controller.rb

```diff
@@ -0,0 +1,41 @@
+# :reek:InstanceVariableAssumption
+# rubocop:disable Style/ClassAndModuleChildren
+class Api::VenuesController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def show
+    @venue = Venue.find(params[:id])
+    render json: @venue
+  end
+
+  def create
+    @venue = Venue.new(venue_params)
+    if @venue.save
+      render json: @venue, status: :created
+    else
+      render json: @venue.errors, status: :unprocessable_entity
+    end
+  end
+
+  def update
+    @venue = Venue.find(params[:id])
```

Comment:
please add a private method 
```
def venue
  @venue ||= Venue.find_by(id: params[:id])
end
```
and just use it whenever it needed

Reply:
i've moved it into 
```ruby
before_action :set_venue
```